### PR TITLE
set tensorstore settings in cloudtpu to make serialization more robust

### PR DESCRIPTION
**Pull request by dlwh**
@yashk2810 sorry for being slow

### CODE REVIEW

File: jax/experimental/array_serialization/serialization.py [hunk 1]
```diff
@@ -63,9 +63,6 @@ class BarrierTimeoutException(Exception):
     "Suggestions for possible fixes:\n"
     "* Check the logs to see if one or more processes failed.\n"
     "* Make sure the training and checkpointing endpoints are close geographically.\n"
-    "* Try setting these environment variables: "
```
Comment (yashk2810) [inline jax/experimental/array_serialization/serialization.py:66]: Let's keep this just in case?

Comment (dlwh) [reply jax/experimental/array_serialization/serialization.py:66]: it'll be a bit weird if it's already set. should I sniff those out and change the error message if they're already set to something? (fwiw, Apple sets the bytes one to 1024 I think)

Comment (yashk2810) [reply jax/experimental/array_serialization/serialization.py:66]: No, it's fine. let's get this in.

**Review by yashk2810 — APPROVED** [review]

File: jax/experimental/array_serialization/serialization.py (hunk 1 above)
Comment (dlwh) [reply jax/experimental/array_serialization/serialization.py]: reverted

Comment (yashk2810) [reply jax/experimental/array_serialization/serialization.py]: Ohh why?

Comment (dlwh) [reply jax/experimental/array_serialization/serialization.py]: i thought that's what you wanted me to do? I can revert the reversion

Comment (yashk2810) [reply jax/experimental/array_serialization/serialization.py]: Yeah.. I meant that let's go with your original CL.

Comment (dlwh) [reply jax/experimental/array_serialization/serialization.py:66]: the reverters have been reverted

**Pull request merged**